db/migrate/20171018141030_create_wb_extra_country_data.rb

```diff
@@ -0,0 +1,11 @@
+class CreateWbExtraCountryData < ActiveRecord::Migration[5.1]
+  def change
+    create_table :wb_extra_country_data do |t|
+      t.references :location, foreign_key: true
+      t.string :year
+      t.string :GDP
```

Comment:
Please use all lowercase letters for consistency all with other tables (even though gdp is an acronym)